docs/source/conceptual/safetensors.md

```diff
@@ -0,0 +1,7 @@
+# Safetensors
+
+Safetensors is a model serialization format for deep learning models. It is [faster](https://huggingface.co/docs/safetensors/speed) and safer compared to other serialization formats like pickle (which is used under the hood in many deep learning libraries). 
+
+TGI depends on safetensors format mainly to enable [tensor parallelism sharding](./tensor_parallelism). For a given model repository during serving, TGI looks for safetensors weights. If there are no safetensors weights, TGI converts the PyTorch weights to safetensors format. 
```

Comment:
Is safetensors integration mainly for tensor parallelism?

Reply:
And loading speed. But it's essential only for TP (we can load TP weights MUCH faster with safetensors)